extensions/sinks/influx/influx.go

```diff
@@ -119,7 +122,7 @@ func (m *influxSink) Open(ctx api.StreamContext) (err error) {
 
 func (m *influxSink) Collect(ctx api.StreamContext, data interface{}) error {
 	logger := ctx.GetLogger()
-	if m.hasTransform {
+	if m.hasTransform || len(m.fields) > 0 {
```

Comment:
Is it possible to not serialize/deserialize? Maybe do it in another PR

Reply:
Yeah, I'll optimize it later.